src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -310,41 +400,15 @@ Follow our [date and time formatting](/content/grammar#date-and-time) guidelines
 
 ### Rows and cells
 
-![Row and Cell States](figma:patterns-tables-basic-formatting-row-cell-states.png)
-
-<table>
-  <thead>
-    <tr>
-      <th width="30%">Row/Cell state</th>
-      <th>Additional guidance</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>1. Empty cell</td>
-      <td>Cells with no values should be left blank. Do not use a hyphen, en, or em dash.</td>
-    </tr>
-    <tr>
-      <td>2. Hover</td>
-      <td></td>
-    </tr>
-    <tr>
-      <td>3. Focused</td>
-      <td></td>
-    </tr>
-    <tr>
-      <td>4. Selected</td>
-      <td></td>
-    </tr>
-    <tr>
-      <td>5. Focused and selected</td>
-      <td>
-        The last row checked is the focused row, unless the user clicks on a different row or uses
-        the keyboard to change focus.
-      </td>
-    </tr>
-  </tbody>
-</table>
+![Row and cell states](figma:patterns-tables-basic-formatting-row-cell-states.png)
+
+| <b>Row/cell state</b>   | <b>Additional guidance</b>                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| 1. Empty cell           | Cells with no values should be left blank. Do not use a hyphen, en, or em dash.                                          |
```

Comment:
omg 🙌 finally. @smritimadan let's remember a follow-on TODO to cleanup outmoded blank cells in component [prop sheets](https://garden.zendesk.com/components/table#table).